powershellgroup/Tests/PSTestModule/DscResources/TestPSRepository/TestPSRepository.psm1

```diff
@@ -21,7 +21,7 @@ function Get-TargetResource {
         PublishLocation           = $null
         ScriptPublishLocation     = $null
         InstallationPolicy        = $null
-        PackageManagementProvider = $null
+        PackageManagementAdapter = $null
```

Comment:
I don't think this value should probably be changed - it doesn't seem related to the DSC resource adapter concept. Recommend reverting the changes to this file, the MOF, and the pester test.